orc8r/cloud/helm/orc8r/values.yaml

```diff
@@ -52,6 +52,34 @@ secret:
     orc8r: orc8r-secrets-configs-orc8r
   envdir: orc8r-secrets-envdir
 
+# certs sub-chart configuration.
```

Comment:
@hcgatewood This probably obsoletes the secret section above right ?

Reply:
No, we are not using any of these auto-generated secrets inside pods.